src/test/java/io/redis/examples/Cmds_string.java

```diff
@@ -0,0 +1,48 @@
+// EXAMPLE: cmds_string
+// REMOVE_START
+package io.redis.examples;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+// REMOVE_END
+// HIDE_START
+import redis.clients.jedis.UnifiedJedis;
+// HIDE_END
+
+// HIDE_START
+public class Cmds_string {
```

Comment:
1. The class name must end with `Example`.
2. We avoid putting underscore (`_`) in class name.